migration-engine/core/src/tests/test_harness/test_api.rs

```diff
@@ -207,14 +203,11 @@ impl TestApi {
     }
 
     pub async fn describe_database(&self) -> Result<SqlSchema, anyhow::Error> {
-        use failure::ResultExt;
-
         let mut result = self
             .describer()
             .describe(self.connection_info().unwrap().schema_name())
             .await
-            .context("Description failed")
-            .compat()?;
+            .expect("Description failed");
```

Comment:
can't use anyhow::Context here because the sql schema describer doesn't return a std::error::Error (it's a failure)